Anki_App/public/public/cards.html

```diff
@@ -22,17 +22,15 @@ <h1>Cards</h1>
         <section class="collections">
             <ul class="collections-list">
                 <li><a href="card.html"><div class="collections-list-item"><h2>+</h2></div></a></li>
-                <li><a><div class="collections-list-item"><h4>Card name</h4><button class="edit-button">Edit</button><button class="delete-button">Delete</button></div></a></li>
-                <li><a><div class="collections-list-item"><h4>Card name</h4><button class="edit-button">Edit</button><button class="delete-button">Delete</button></div></a></li>
-                <li><a><div class="collections-list-item"><h4>Card name</h4><button class="edit-button">Edit</button><button class="delete-button">Delete</button></div></a></li>
-                <li><a><div class="collections-list-item"><h4>Card name</h4><button class="edit-button">Edit</button><button class="delete-button">Delete</button></div></a></li>
-                <li><a><div class="collections-list-item"><h4>Card name</h4><button class="edit-button">Edit</button><button class="delete-button">Delete</button></div></a></li>
-                <li><a><div class="collections-list-item"><h4>Card name</h4><button class="edit-button">Edit</button><button class="delete-button">Delete</button></div></a></li>
-                <li><a><div class="collections-list-item"><h4>Card name</h4><button class="edit-button">Edit</button><button class="delete-button">Delete</button></div></a></li>
-                <li><a><div class="collections-list-item"><h4>Card name</h4><button class="edit-button">Edit</button><button class="delete-button">Delete</button></div></a></li>
-                <li><a><div class="collections-list-item"><h4>Card name</h4><button class="edit-button">Edit</button><button class="delete-button">Delete</button></div></a></li>
-                <li><a><div class="collections-list-item"><h4>Card name</h4><button class="edit-button">Edit</button><button class="delete-button">Delete</button></div></a></li>
-                
+                <li><div class="collections-list-item"><h4>Card name</h4><button class="edit-button">Edit</button><button class="delete-button">Delete</button></div></a></li>
```

Comment:
Markup is invalid now.

Reply:
In this page a links was unnecessary. 
They was added when i copied markup from collections page.

Reply:
Oh, sorry. Find the misstake. Fixed it.